templates/prWaitingRoom.html

```diff
@@ -154,23 +154,23 @@ <h2 class="mdc-dialog__title" id="my-dialog-title">Join room</h2>
     </div>
     <div class="mdc-dialog__scrim"></div>
 </div>
-<script src="app.js"></script>
+<script src="../FirebaseRTC/public/app.js"></script>
```

Comment:
Similar to the previous comment, update this src to match the new naming system. The project directory is the root.